base/fft/FFTW.jl

```diff
@@ -101,16 +101,16 @@ alignment_of(A::FakeArray) = Int32(0)
 # FFTW's api/import-wisdom-from-file.c file].
 
 function export_wisdom(fname::AbstractString)
-    f = ccall(:fopen, Ptr{Void}, (Cstring,Ptr{UInt8}), fname, "w")
+    f = ccall(:fopen, Ptr{Void}, (Cstring,Cstring), fname, "w")
```

Comment:
Note that `Ptr{UInt8}` is safe for things like this because we know that the literal string `"w"` contains no embedded NUL chars.  So in my initial `Cstring` PR I left these as-is as an optimization (albeit probably pointless).

It looks like just about all of the `Ptr{UInt8}`-to-`Cstring` substitutions that you made in argument lists are of this form.